strapi/strapi
# feat: reordering for polymorphic relations in Content API

### What does it do?

- adds support for set, connect, and disconnect through the Content API and document service for polymorphic relations
  - includes support for re-ordering

In a later PR we should refactor the massive relations test files into multiple smaller files

### Why is it needed?

They were buggy and incomplete on v4 and absent from v5; this adds support for them in v5, including reordering

### How to test it?

API tests have been added

Manually, ensure that submitting an API request to set, connect, and disconnect polymorphic relations works as expected, especially the case described in issue 20961

### Related issue(s)/PR(s)

Fixes https://github.com/strapi/strapi/issues/20961

Should fix? https://github.com/strapi/strapi/issues/20931

v4 PR: https://github.com/strapi/strapi/pull/21135

DX-1545

DX-1726
